util/test/paratest.chapcs

```diff
@@ -110,30 +107,12 @@ def get_num_non_exclusive_nodes():
     return num_non_exclusive_nodes
 
 
-def get_num_oversub_jobs():
-    """
-    Get the number of jobs that allow oversubscription (permit node sharing)
-    """
-    squeue_cmd = ['squeue',
-                  '--partition={0}'.format(get_partition()),
-                  '--noheader',
-                  '--format="%h"']
-    squeue_out = [s.upper() for s in run_command_wrapper(squeue_cmd)]
-    return squeue_out.count('YES')
-
-
 def get_good_nodepara():
     """
-    Get a "good" nodepara value: default to 4 for comm=none testing, and
-    3 for comm!=none since that's already oversubscribed. If no other shared
-    jobs are running bump the nodepara by 1.
+    Get a "good" nodepara value. Currently defaults to 6, but could differ
+    based on parition, comm, or whether other jobs are running.
```

Comment:
"could differ" is a bit vague here. Perhaps something like "Currently always 6. We could adjust it based on [.....]."